dataflow/build.gradle

```diff
@@ -105,6 +110,14 @@ def testDataflowAnalysis(taskName, dirName, className) {
 
 testDataflowAnalysis("liveVariableTest", "live-variable", "livevar.LiveVariable")
 
+
+// allDataflowTests for dataflow analysis tests
+tasks.create(name: 'allDataflowTests', group: 'Verification') {
+    description 'Run all dataflow analysis tests'
+    // Once there is a new type of dataflow test, add it into dependsOn
+    dependsOn('liveVariableTest')
```

Comment:
Is there a way to create and add a new task to dependency here instead of having to write both `testDataflowAnalysis` and `dependsOn`?

Reply:
I guess the easiest way is to declare that `allDataflowTests` depends on the new task in `testDataflowAnalysis`.